### PR TITLE
update package.json of Jquery countdown

### PR DESCRIPTION
**Pull request by maruilian11**
@Amomo  could you please help me check this PR for #6526 ? thank you!

### CODE REVIEW

File: ajax/libs/jquery-countdown/package.json [hunk 1]
```diff
@@ -5,12 +5,32 @@
   "description": "Countdown for jQuery.",
   "homepage": "http://keith-wood.name/countdown.html",
   "keywords": [
-    "jquery",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-countdown/package.json]: why remove jquery?

File: ajax/libs/jquery-countdown/package.json [hunk 2]
```diff
@@ -5,12 +5,32 @@
...
+    "name": "Keith Wood",
+    "url": "http://keith-wood.name/"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kbwood/countdown.git",
+    "basePath": "",
+    "files": [
+      "jquery.countdown*.js",
+      "jquery.countdown*.css",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-countdown/package.json]: use easier method ... `"jquery.countdown*.+(css|js)"`

File: ajax/libs/jquery-countdown/package.json [hunk 3]
```diff
@@ -5,12 +5,32 @@
...
+      "jquery.plugin*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-countdown/package.json]: `"jquery.+(plugin|countdown)*.+(css|js)"`

**Comment by maruilian11** [comment]
@Amomo  could you please help me check this PR? thank you!

**Pull request merged**